### DISCUSSION TITLE
Gatsby and Twitter Cards

### DISCUSSION THREAD

**Discussion by rupertxyz** [gatsbyjs/gatsby#37625] [Help] [answered]
Hi everyone,

Twitter Cards are a paiiiin. Can someone help me figure out why the large image isn't shown on my blog posts? Here an example post: https://docsautomator.co/blog/why-docsautomator/

I've set up a SEO component like so: https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-seo-component/

Meta tags / head in general is identical with the landing page, for which a large image is shown.

Thanks for the help!

**Comment by violy** [comment]
After share your link, I have the Twitter card with image? what did you expect? see capture below 👇

![image](https://user-images.githubusercontent.com/881828/217751806-3aead588-3f6f-48a5-bec8-b2abf0aba97e.jpeg)

**Comment by rupertxyz** [comment, marked as answer]
Thanks for your reply. I managed to solve it yesterday by adding all my blog cover images to my static folder instead of serving them from a source folder.
